src/cst/matchers.rs

```diff
@@ -4,6 +4,6 @@ use libcst_native::Module;
 pub fn match_module(module_text: &str) -> Result<Module> {
     match libcst_native::parse_module(module_text, None) {
         Ok(module) => Ok(module),
-        Err(_) => return Err(anyhow::anyhow!("Failed to extract CST from source.")),
+        Err(_) => Err(anyhow::anyhow!("Failed to extract CST from source.")),
```

Comment:
Not sure why CI doesn't catch this, but I got this warning on my machine:

```
> cargo clippy
    Checking ruff v0.0.101 (/home/haru/Desktop/repositories/ruff)
warning: unneeded `return` statement
 --> src/cst/matchers.rs:7:19
  |
7 |         Err(_) => return Err(anyhow::anyhow!("Failed to extract CST from source.")),
  |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: remove `return`: `Err(anyhow::anyhow!("Failed to extract CST from source."))`
  |
  = note: `#[warn(clippy::needless_return)]` on by default
  = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_return

warning: `ruff` (lib) generated 1 warning
    Finished dev [unoptimized + debuginfo] target(s) in 1.51s
```

Reply:
I think it's because CI is using nightly (`cargo +nightly clippy --all`), I see the same behavior locally when toggling between nightly and stable. I don't understand why though.